diff --git a/TSOClient/tso.world/World.cs b/TSOClient/tso.world/World.cs
index c2b1e7f7..1a482828 100644
--- a/TSOClient/tso.world/World.cs
+++ b/TSOClient/tso.world/World.cs
@@ -571,7 +571,7 @@ namespace FSO.LotView
 
             if (Blueprint != null)
             {
-                if (!Content.Content.Get().TS1) Blueprint.Weather?.Update();
+                if (Content.Content.Get().TS1) Blueprint.Weather?.Update();
                 var partiCopy = new List<ParticleComponent>(Blueprint.Particles);
                 foreach (var particle in partiCopy)
                 {
